vm/src/obj/objlist.rs

```diff
@@ -409,6 +409,26 @@ fn list_pop(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     }
 }
 
+fn list_remove(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(
+        vm,
+        args,
+        required = [(list, Some(vm.ctx.list_type())), (needle, None)]
+    );
+
+    let mut elements = get_mut_elements(list);
+    let i = elements
+        .iter()
+        .position(|r| objbool::get_value(&vm._eq(needle.clone(), r.clone()).unwrap()));
+    if i.is_some() {
+        elements.remove(i.unwrap());
```

Comment:
While this works fine, it's more idiomatic in rust to avoid this unwrap by using an `if let`, e.g.:

```rust
if let Some(index) = i {
    elements.remove(index);
} else {
    ...
}
```

Reply:
Done.